flexmeasures/data/models/planning/tests/conftest.py

```diff
@@ -145,15 +145,15 @@ def inflexible_devices(db, building) -> dict[str, Sensor]:
         name="PV power sensor",
         generic_asset=building,
         event_resolution=timedelta(hours=1),
-        unit="kW",
+        unit="MW",
```

Comment:
I was expecting a test sensor to be updated from MW to kW, actually. Why is it the opposite?

Reply:
This is because before this change was introduces, any power value was interpreted to have `MW` units. Now, we had to change it here to reflex that. Alternatively, I can scale the values to be in `kW`.

Reply:
Ah okay. Yes, covering the conversion in a test would be good to see.

Reply:
Absolutely! I've change one of the inflexible devices to use kW, maintaining the previous value.